File: BackEnd/src/entity/OrderDetail.java
package entity;

public class OrderDetail {
    private String itemCode;
    private String orderId;
    private int  qty;
    private double price;

    public OrderDetail() {
    }

    public OrderDetail(String itemCode, String orderId, int qty, double price) {
        this.setItemCode(itemCode);
        this.setOrderId(orderId);
        this.setQty(qty);
        this.setPrice(price);
    }

    public String getItemCode() {
        return itemCode;
    }

    public void setItemCode(String itemCode) {
        this.itemCode = itemCode;
    }

    public String getOrderId() {
        return orderId;
    }

    public void setOrderId(String orderId) {
        this.orderId = orderId;
    }

    public int getQty() {
        return qty;
    }

    public void setQty(int qty) {
        this.qty = qty;
    }

    public double getPrice() {
        return price;
    }

    public void setPrice(double price) {
        this.price = price;
    }
}

File: FrontEnd/model/OrderDetail.js
function OrderDetail(itemCode,price,oQty) {
    var _itemCode = itemCode;
    var _price = price;
    var _oQty = oQty;


    this.setItemCode = function (code) {
        _itemCode = code;
    }
    this.setPrice = function (price) {
        _price = price;
    }
    this.setOQty = function (oQty) {
        _oQty=oQty;
    }

    this.getItemCode = function () {
        return _itemCode;
    }
    this.getPrice = function () {
        return _price;
    }
    this.getOrderQty = function () {
        return _oQty;
    }
}